script/Ciel/Day 8/08_01_CIEL08_2_1.txt

```diff
@@ -1,27 +1,29 @@
 [sha:7cc854c20b617d1112edc84dfdacd9a2394486fa]{
 -- Page 10007, Offset 18704.
 -- ……怪しい。あの購買部好き……というか特定のパン好き……の先輩が、わざわざ外にランチしに行くとは思えない。
-...Very suspicious. I don't think that my school store loving, or rather special bread loving, Senpai would deliberately go out to have lunch.
+// god save me
+...Suspicious. My cafeteria loving Senpai, or rather, my specifically one kind of bread loving Senpai wouldn't go out of her way to have lunch somewhere outside school.
 }
 [sha:25602e2a6c002d27c6e9c4ba96d260a4a3904683]{
 -- Page 172, Offset 18705.
 -- もしや、保健室で言っていた“吸血鬼のねぐら”を調査しに行くのではないだろうか？
-Isn't she rather going to investigate that vampire roost she talked about in the infirmary, perchance?
+// was ねぐら=roost decided on?
```

Comment:
I hope not. "roost" is pretty terrible. In day 7 we used "nest" which I think is much better language. The Church especially uses "vermin-like" language to describe vampires and vampirings

Reply:
originally had this as nest but changed it back. i'll go ahead and change the other instances of 'roost' in d8 then